src/EnergyPlus/DualDuct.cc

```diff
@@ -1975,17 +1747,29 @@ namespace DualDuct {
             return;
         }
 
-        if (this->OAPerPersonMode == PerPersonMode::DCVByCurrentLevel) {
+        bool UseOccSchFlag; // TRUE = use actual occupancy, FALSE = use total zone people
+        bool PerPersonNotSet;
+        switch (this->OAPerPersonMode) {
```

Comment:
I think we've covered this before, but ... don't use control flow to set boolean variables, just set the variables directly to the control flow logic.